src/commands/cna/test.js

```diff
@@ -23,25 +20,12 @@ class CNATest extends CNABaseCommand {
     // some things we could do here:
     // test configurations, ie remote-actions deployed and called from local
     // this just runs package.json scripts.test, we could also check that this is in fact a cna project
-
-    const spinner = ora()
-
-    const taskName = flags.e2e ? 'e2e tests' : 'unit tests'
     const command = flags.e2e ? 'e2e' : 'test'
-
-    this.log(chalk.bold(`> ${taskName}`))
-    spinner.start(taskName)
-
     try {
-      await cnaHelper.runPackageScript(command, process.cwd(), { silent: true })
+      await cnaHelper.runPackageScript(command, process.cwd(), { silent: !flags.verbose })
     } catch (e) {
-      this.log()
-      spinner.fail(chalk.bold(chalk.red(`${taskName} failed !`)))
-      return this.error(e, { exit: e.exitCode })
+      return this.error(e.message, { exit: e.exitCode })
```

Comment:
some tests are failing because they expect to get an error and not an error.message. 
Did you do the e.message change on purpose?